sdk/servicebus/service-bus/migrationguide.md

```diff
@@ -108,11 +108,11 @@ brings this package in line with the [Azure SDK Design Guidelines for Typescript
   ruleManager.removeRule();
   ```
 
-* createSender() and createSessionReceiver() are now async methods and initialize the connection
+* createSessionReceiver() is now an async method and initialize the connection
 
-Prior to v7 `createSender()` and `createSessionReceiver()` worked using lazy-initialization, where the
+Prior to v7 `createSessionReceiver()` worked using lazy-initialization, where the
 AMQP connection would only be initialized on first send or receiving of a message.
```

Comment:
```suggestion
the receiver link to the session is only initialized when any of the async methods on the `SessionReceiver` is first called
```

Reply:
Slight modification (I believe the tense should still be past, not present).